main.js

```diff
@@ -21,4 +177,115 @@ navLink.addEventListener('click', () => {
     document.body.style.overflowY = 'scroll';
     menu.classList.toggle('show-menu');
   });
+
+ 
+
+  const cardWorks=document.querySelector('.card-works');
+  const topDesktop=document.createElement('div');
+  topDesktop.classList.add('top-desktop');
+  cardWorks.appendChild(topDesktop);
+
+  const headingPart=document.createElement('div');
+  headingPart.classList.add('heading-part');
+  topDesktop.appendChild(headingPart);
+
+  const headingOne=document.createElement('h1');
+  headingOne.textContent=ProjectDetail[1].title;
+  headingPart.appendChild(headingOne);
+
+  
+  const headingTwo=document.createElement('h2');
+  headingTwo.classList.add('desktopHeading')
+  headingTwo.textContent=ProjectDetail[1].popTitle;
+  headingPart.appendChild(headingTwo);
+
+  const closeIcon=document.createElement('p');
+  closeIcon.classList.add('fa-times');
+  closeIcon.classList.add('para');
```

Comment:
- [ ] I see that you are using the letter x as a close button but you should stick to the Figma design and use the icon from Figma so I encourage you to do that

- [ ] Figma 
![image](https://user-images.githubusercontent.com/61790587/168917425-b049c531-faec-4411-b7e9-7beaef4e86d2.png)

- [ ] Yours 
![image](https://user-images.githubusercontent.com/61790587/168917445-7de0efec-f2e9-4888-96c9-2e1af58036f2.png)